freetextresponse/freetextresponse.py

```diff
@@ -0,0 +1,437 @@
+'''
+This is the core logic for the FreeText XBlock
+'''
+
+import os
+import re
+
+import pkg_resources
+
+from enum import Enum
+from xblock.core import XBlock
+from xblock.fields import Scope
+from xblock.fields import String
+from xblock.fields import Integer
+from xblock.fields import Float
+from xblock.fragment import Fragment
+
+
+class Freetextresponse(XBlock):
+    #  pylint: disable=too-many-ancestors, too-many-instance-attributes
```

Comment:
Where does the `too-many-ancestors` error come from, the classes from which `XBlock` is derived?